<launch>

  <arg name="imame_raw_topic" default="kinect/rgb/image_raw"/>

  <node name="image_process" pkg="object_recognition" type="image_process" output="screen">
    <remap from="/usb_cam/image_raw" to="$(arg imame_raw_topic)" />
  </node>

</launch>